### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by bkeepers** [chordbook/chordbook#336] [General]
Use this thread to say hi and introduce yourself to other contributors.  What's your story, what brings you here? What are you hoping to get out of or contribute to Chord Book?

I'll kick it off…

**Comment by bkeepers** [comment]
👋 Hey, I'm Brandon and I am the creator of Chord Book. I am an amateur musician and a professional software developer. I started playing guitar in high school and played for fun with friends and at church. I have also dabbled with mandolin and ukulele, but am even less proficient. My commitment to playing consistently has ebbed and flowed over the last 25 years, and while I am little more than a mediocre musician, I still love playing.

I created Chord Book because I want a better way to discover and learn new songs. I believe music is inherently communal, and our song book should be too. I find the existing guitar tab sites lacking and finally got fed up with them. So I started Chord Book and am excited to see where it goes.

**Reply by edwinzap** [reply]
Your work is really nice! Is there any way we could talk together someday?

**Comment by edwinzap** [comment]
Hi, I'm Miguel. I'm a professionnal software developer and musician (I'm a guitar teacher).

We created a few years ago (during covid pandemic) [ChordProject](https://chordproject.com/) mostly for the same reasons as @bkeepers. We couldn't find at the time any good website that handles correctly the chordpro format. For this project we started to build [some libraries](https://github.com/chordproject), often based on other libraries, because no library had the features and the quality we were searching for. Unfortunately, we stopped developing the project for some personal reasons.

Few weeks ago, we decided to continue developing the project. While searching for similar websites, I found this repo and I think we could collaborate!

**Reply by jrmora** [reply]
Absolutely @bkeepers . We share the same idea and similar codes with [ChordProject](https://chordproject.com/editor/bEZvFYO7VtAFpfXQef9w) . We should talk. Let's strike the Chords together!

![image](https://github.com/chordbook/chordbook/assets/9900351/12138fe9-c51b-4903-84fd-cc80ff811d21)

**Reply by bkeepers** [reply]
:wave: Hey @edwinzap and @jrmora, thanks for reaching out! I'd love to talk about ways to collaborate. Send me an email at brandon@opensoul.org and we can coordinate a time to talk?

**Reply by edwinzap** [reply]
Hi, I sent you an email a week ago, did you see it? Thank you!

**Reply by bkeepers** [reply]
@edwinzap no, I didn't get anything. I just searched my email and am not seeing anything.

**Reply by edwinzap** [reply]
I just sent it again. I hope you get it :)

**Reply by bkeepers** [reply]
Still didn’t get anything. What’s your email address and I’ll try sending you something.

**Reply by bkeepers** [reply]
Nevermind. Found it in my spam folder

**Comment by jrmora** [comment]
Hi Brandon,

Yesterday ,I sent you a google meets invitation for tomorrow (friday 11h00)
I hope you received it 🙂

Have a nice day


Jaime
________________________________
De: Brandon Keepers ***@***.***>
Enviado: domingo, 10 de marzo de 2024 7:49 a. m.
Para: chordbook/chordbook ***@***.***>
Cc: jrmora ***@***.***>; Mention ***@***.***>
Asunto: Re: [chordbook/chordbook] Introductions (Discussion #336)


Nevermind. Found it in my spam folder

**Reply by bkeepers** [reply]
I don't see an invite in my inbox or spam folder.

**Reply by jrmora** [reply]
Send it again to brandon@opensoul.org  :)
